<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="LAPTOP-UGQAOAUR" failures="1" tests="1" name="Default test(failed)" time="12.969" errors="0" timestamp="19 Apr 2020 14:28:48 GMT">
  <testcase classname="com.app.Tests.SignOnTest" name="ValidateSignOn" time="1.933">
    <failure type="java.lang.AssertionError" message="expected [http://newtours.demoaut.com/mercurysignon.php] but found [http://newtours.demoaut.com/mercurysignon.php?osCsid=763d88e5e7b23c04085df8704b34b7ab]">
      <![CDATA[java.lang.AssertionError: expected [http://newtours.demoaut.com/mercurysignon.php] but found [http://newtours.demoaut.com/mercurysignon.php?osCsid=763d88e5e7b23c04085df8704b34b7ab]
at com.app.Tests.SignOnTest.ValidateSignOn(SignOnTest.java:71)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- ValidateSignOn -->
  <testcase classname="com.app.Tests.SignOnTest" name="ValidateWebElementPresent" time="0.0">
    <ignored/>
  </testcase> <!-- ValidateWebElementPresent -->
</testsuite> <!-- Default test(failed) -->
